pkg/kopia/command/common.go

```diff
@@ -40,12 +58,26 @@ func stringSliceCommand(args logsafe.Cmd) []string {
 func commonArgs(cmdArgs *CommandArgs) logsafe.Cmd {
 	c := logsafe.NewLoggable(kopiaCommand)
 
-	if cmdArgs.LogLevel != "" {
-		c = c.AppendLoggableKV(logLevelFlag, cmdArgs.LogLevel)
+	logLevel := cmdArgs.LogLevel
```

Comment:
do you think this requires change in the consumers of this function for example `SnapshotCreate` etc? Are we planning to do that in other PR?

Reply:
I think it is not required. 

Behavior before change:
Either level from var OR error

Behavior after change:
Either level from var **OR level from env** OR error

Taking level from env is needed only when debugging. 
I went through the invokers and LogLevel is not set there. But if it will be set there to some value, I believe, decision of priority between env and their value should be done on their side.